### ISSUE TITLE
Odd shadow seemingly caused by underwater effects.

### ISSUE THREAD

**Issue by andrew-sidereal** [crest-ocean/crest#636]
**Describe the bug**
These screenshots show it well: https://imgur.com/a/VWNMcl9
If I don't use the underwater components on the camera, then there is no problem.
When I enable the underwater components, the underwater effect works, but this odd shadow appears infront of the camera, then disappears... then reappears. It doesn't show up on the ground, or if I'm further up away from the water.

If, in game, I turn both UnderWaterCurtainGeom and UnderWaterMeniscus off, the issue ceases. If I turn EITHER of them on, then it resumes. 

Built-in Rendering Pipeline. Unity 2019.4.4.

**Comment by andrew-sidereal** [comment]
Here's a video, in case it helps: https://drive.google.com/file/d/1UmdLqVtTD0ui95Bp87h8fxvdKIIbOUwp/view?usp=sharing

**Comment by daleeidd** [comment]
Thanks. Would you be able to provide a screenshot using the ocean debug GUI with the shadow data showing? Here is an example:

![1](https://user-images.githubusercontent.com/5249806/91511977-c91b4480-e895-11ea-827a-30be912f76cb.jpg)

**Comment by andrew-sidereal** [comment]
https://i.imgur.com/u5D9qTO.jpg

**Comment by andrew-sidereal** [comment]
Actually, here's what you're looking for. Definitely shows up on the Shadow: https://i.imgur.com/JYarIOO.jpg

**Comment by daleeidd** [comment]
Thanks. Hmmm.

Does it go away if you set shadow cascades to zero?
Can you try disabling shadows on both the mesh renderers on the underwater game objects? (I don't think this will make a difference but worth a try)
Have you tried disabling Enviro to see if there might be a strange integration issue?

**Comment by andrew-sidereal** [comment]
Well, I suspected Enviro, so I uninstalled it entirely. I also got a fresh copy of Crest from master, re-installed it. Alas! The issue remained. Then I tried disabling the shadows on the mesh renderers for the underwater game objects. I had little hope this would work, but it DID! The issue seems to go away after this change: https://i.imgur.com/ZF1oVSt.png

The only other shaders I'm using at this point is MicroSplat. Maybe it's related somehow? Don't know.

Happy it's resolved!
